fastlane/Fastfile

```diff
@@ -189,31 +189,40 @@ platform :ios do
     sh "git push origin --tags"
   end
 
-  desc "Add a tag for the current version number push to repo."
+  desc "Add a release tag for the latest beta and push to repo."
   lane :tag_release do |options|
```

Comment:
I wonder if it'd be nice to have an optional `sha` to pass in here that takes precedence over `get_latest_build_number()` that we could send in through the Jenkins job. With your work on [https://phabricator.wikimedia.org/T275167](https://phabricator.wikimedia.org/T275167) I could see a situation where the latest beta has been bumped up (due to us working ahead to the next release and merging things) before Apple approves the current release, so the wrong commit gets tagged.

But I'm also fine with waiting until we run into that situation first before adding complexity.

Reply:
Done. And jenkins has a spot to put it in. If you leave it blank, it uses the last beta. Otherwise, you can put any valid tag or commit hash in. And invalid tags/commit hashes will cause the process to fail.

I also ran `bundle update` to update the gems.